RealTimeSimulator.html

```diff
@@ -1,2682 +0,0 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-  <title>Group Income Simulator</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
-  <link rel="icon" type="image/svg+xml" href="favicon.ico">
-  <link rel="stylesheet" href="style.css">
-</head>
-
-<body>
-  <div class="gis_layout">

```

Comment:
This file is very important as it contains the version of the simulator that demonstrates the real-time algorithm (that we never ended up using). We definitely do not want to delete it.

Reply:
Ok, will add it back then. (noticed this is used in nowhere so deleted)